custom-metrics-stackdriver-adapter/pkg/adapter/provider/translator.go

```diff
@@ -137,6 +161,44 @@ func (t *Translator) GetRespForMultipleObjects(response *stackdriver.ListTimeSer
 	return t.metricsFor(values, groupResource, metricName, list)
 }
 
+// GetRespForExternalMetric translates Stackdriver response to list of External Metrics
+func (t *Translator) GetRespForExternalMetric(response *stackdriver.ListTimeSeriesResponse, metricName string) ([]external_metrics.ExternalMetricValue, error) {
+	if len(response.TimeSeries) < 1 {
+		return nil, provider.NewExternalMetricNotFoundError(metricName)
+	}
+	metricValues := []external_metrics.ExternalMetricValue{}
+	// Find time series with specified labels matching
+	// Stackdriver API doesn't allow complex label filtering (i.e. "label1 = x AND (label2 = y OR label2 = z)"),
+	// therefore only part of the filters is passed and remaining filtering is done here.
```

Comment:
This comments says there should be some filtering here, but I see nothing of the sort. Am I missing something? Or is there some code missing here?